RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -3768,7 +3798,13 @@ boolean auto_change_mcd(int mcd)
 	{
 		return true;
 	}
-	return change_mcd(next);
+	if(immediately)
+	{
+		return change_mcd(next);
```

Comment:
This will get overwritten in pre-adventure to the previous value of `auto_mcd_target`. Do we need to set `auto_mcd_target` here?

Reply:
change made